README.md

```diff
@@ -1 +1,33 @@
-# angular-setup-readme
+# Angular Setup + directory structure
+
+## Overview
+
+We need to ensure that we have a level-headed way of structuring our applications from the beginning, otherwise we could end up with files everywhere, which means it's hard to know exactly what you are editing and where.
+
+## Objectives
+
+- Define the core structure of an Angular application
+- Create a feature based file structure
+
+## Core Structure
+
+Angular supports modules, and our application file structure will reflect this. Each module will have directives, templates, controllers and services.
+
+Like a normal application, we'll have separate folders for our JavaScript, CSS and images. However, inside our JavaScript folder we'll have individual folders for each module in our Angular application. Angular needs at least one module to kick things off, and we're going to call our main module "app".
+
+This means that you'll end up with a directory structure like this -
```

Comment:
It's confusing to see this file structure with a bunch of empty folders. Show an actual, finished app with this structure first and include a note like "don't worry about what all of these files and folders do yet, we'll be discussing that in future lessons"